jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -261,34 +265,57 @@ public class CategoryAttributesDetailView extends StandardDetailView<CategoryAtt
     protected CollectionLoader<TargetViewComponent> targetScreensDl;
     @ViewComponent
     protected InstanceContainer<CategoryAttribute> categoryAttributeDc;
-
+    protected AttributeLocalizationComponent localizationFragment;
+    protected List<TargetViewComponent> targetScreens = new ArrayList<>();
     @ViewComponent("dependsOnAttributesField.clear")
     private ValueClearAction<String> dependsOnAttributesFieldClear;
     @ViewComponent("dependsOnAttributesField.select")
     private MultiValueSelectAction<String> dependsOnAttributesFieldSelect;
     @ViewComponent
     private JmixButton editEnumerationBtn;
-
-    protected AttributeLocalizationComponent localizationFragment;
-
-    protected List<TargetViewComponent> targetScreens = new ArrayList<>();
-
     private boolean isRefreshing = false;
 
+    private final List<String> defaultEnumValues = new ArrayList<>();
+
     @Subscribe
     protected void onInit(InitEvent event) {
         initAttributeForm();
         initCalculatedValuesAndOptionsForm();
         initViewGrid();
     }
+    private void initDefaultEnumField() {
```

Comment:
Let's format more carefully. Here there is no empty lines between methods. After this method there are two empty lines between methods.
Also, all methods in the class are protected. Let's make the new ones protected as well.